release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1376,22 +1376,22 @@ In the table below, features are marked with the following statuses:
 |CSI volume expansion
 |TP
 |TP
-|
+|GA
 
 |CSI AliCloud Disk Driver Operator
 |-
 |GA
 |GA
 
 |CSI Azure Disk Driver Operator
-|TP
+|-
```

Comment:
In the 4.10 RNs, this is listed as TP for 4.9, so I think this change might be wrong